src/comp_loinc/schema/comp_loinc.yaml

```diff
@@ -6,8 +6,9 @@ prefixes:
   loinc_property: http://loinc.org/property/
   owl: http://www.w3.org/2002/07/owl#
   rdfs: http://www.w3.org/2000/01/rdf-schema#
-  sct: http://snomed.info/sct
-  comploinc: http://comploinc/
+  sct: http://snomed.info/sct/
```

Comment:
Ah, great! Will this PR address?:
- #98

We should also make the update to `PREFIXES_METRICS` in the `makefile`, which currently uses `--prefix 'SNOMED: http://snomed.info/'`.